src/alfasim_sdk/_internal/cli.py

```diff
@@ -68,22 +68,28 @@ def new(dst, caption, plugin_id, author_name, author_email):
 
     .. code-block:: bash
 
-        \---myplugin
-        |   CMakeLists.txt
-        |   tasks.py
-        |
-        +---assets
-        |       plugin.yaml
-        |       README.md
-        |
-        \---src
+        <dest>
+        \---<plugin_id>
             |   CMakeLists.txt
-            |   hook_specs.h
-            |   myplugin.cpp
+            |   tasks.py
             |
-            \---python
-                    myplugin.py
-
+            +---assets
+            |       plugin.yaml
+            |       README.md
+            |
+            \---src
+                |   CMakeLists.txt
+                |   hook_specs.h
+                |   <plugin_id>.cpp
+                |
+                \---python
+                    |   <plugin_id>.py
+                    |
+                    \---alfasim_sdk_plugins
+                        \---<plugin_id>
+                                __init__.py
+
+    Any python code placed in `alfasim_sdk_plugins/<plugin_id>` is importable by `import alfasim_sdk_plugins.<plugin_id>.<my_extra_module>`.
```

Comment:
Just a minor change to improve the documentation:
```suggestion
    Any python code placed in ``alfasim_sdk_plugins/<plugin_id>`` is importable by using ``import alfasim_sdk_plugins.<plugin_id>.<my_extra_module>``.
```

Reply:
let's see how sphinx hanble this...